Samples/VSTO-shared-code-migration/README.md

```diff
@@ -5,44 +5,45 @@ products:
 - office-365
 languages:
 - javascript
+- csharp
 extensions:
   contentType: samples
   technologies:
   - Add-ins
-  createdDate: 12/2/2019 1:25:00 PM
+  createdDate: 12/30/2019 1:25:00 PM
 description: ""
 ---
 
 # Use a shared library to migrate your Visual Studio Tools for Office add-in to an Office web add-in
 
-Visual Studio Tools for Office (VSTO) add-ins can only run in Office on Windows. By migrating your code to an Office web add-in, you can expand the reach of your add-in to mobile and online platforms. Migrating the code will involve rewriting it in the JavaScript language, and also updating it to use the Office.js APIs. However, you can save on migration cost by creating a shared library and keeping some of your existing add-in code in its current language.
+![Sharing code between a VSTO Add-in and an Office Web Add-in](shared-code-library-migration.png)
 
-This sample includes a start folder that accompanies the article TBD... If you want to learn how to go through the steps over creating a shared library for an existing VSTO add-in, follow this article.
+Visual Studio Tools for Office (VSTO) add-ins can only run in Office on Windows. By migrating your code to an Office web add-in, you can expand the reach of your add-in to mobile and online platforms. Migrating the code will involve rewriting it in the JavaScript language, and also updating it to use the Office JavaScript APIs. However, you can reduce migration cost by creating a shared library and keeping some of your existing add-in code in its current language.
```

Comment:
Fourth sentence: consider changing "creating a shared library and keeping some" to "creating a shared library to keep some".